isso/js/app/templates/comment.js

```diff
@@ -7,16 +7,19 @@ var html = function (globals) {
   var svg = globals.svg;
 
   var author = comment.author ? comment.author : i18n('comment-anonymous');
+  var isPageAuthor = conf["page-author-hashes"].indexOf(comment.hash) > -1;
+  var pageAuthorClass = (isPageAuthor ? " isso-is-page-author" : '');
+  var pageAuthorSuffix = (isPageAuthor ? " (Author)" : '');
```

Comment:
This is a string that must be translated. Add it to `i18n/en.js` (and - in a separate commit - to any other translation files of languages you speak)

~~Also, I think we might have a problem with right-to-left languages. With e.g. `Amir` + ` (Author)` in Arabic (`أمير` and `(مؤلف)`, this should be `أمير (مؤلف)`~~ Huh, Javascript apparently already does the correct thing for us and keeps the Arabic part RTL, but the latin part ltr. Nice.

Reply:
Also, the ` (Author)` part should have a different color. Otherwise, it wouldn't stick out

And everyone could just type in `BBao (Author)` into the comment box - you'd probably catch it when manually unlocking comments, but still annoying for people without moderation turned on.

Reply:
This is Farsi with a Farsi name ("Farid"-> "فرید"):    `فرید•نویسنده•17 دقیقه پیش`
This is Farsi with a Latin-script name ("John"):         `John•نویسنده•17 دقیقه پیش`

"Author" ("نویسنده") will confusingly be placed at the rightmost edge if the Author name is Latin-script. "نویسنده" highlighted in red for this screenshot:

![Screenshot from 2022-05-07 19-24-05](https://user-images.githubusercontent.com/10212877/167265264-a3cf93ac-f947-4fd3-b52a-cf3981eef3a0.png)

Not all that important for this PR, we'd probably be better off with proper RTL support by dynamically setting `direction: rtl` for all of `#isso-thread` if detected/set language is an RTL one.